### PR TITLE
task_week_2

### PR DESCRIPTION
**Pull request by MinaKamil**

### CODE REVIEW

File: week_2/practice/src/pages/todo.js [hunk 1]
```diff
@@ -61,22 +62,29 @@ export default class TodoPage extends Component {
     /* to simulate an API call */
     this.addMockData();
   }
+  handleCompleted = () => {
```
Comment (3ba2ii) [inline week_2/practice/src/pages/todo.js:65]: Great job handling toggle isCompleted flag state 👏🏻

File: week_2/practice/src/components/TodoList.js [hunk 2]
```diff
@@ -5,15 +5,25 @@ export default class TodoList extends Component {
   render() {
     return (
       <section className='todo-section-container'>
+        <button onClick={ this.props.handleCompleted }>Show Completed</button>
```
Comment (3ba2ii) [inline week_2/practice/src/components/TodoList.js:8]: You also did a great job here with the toggle button but I encourage you to update the button text based on the current `isCompleted` flag  just for a better UX
``` javascript
<button onClick={ this.props.handleCompleted}>
     {this.props.isCompleted ? 'Show All' : 'Show Completed' }
</button>
```

File: week_2/practice/src/components/TodoList.js [hunk 3]
```diff
@@ -5,15 +5,25 @@ export default class TodoList extends Component {
...
         <ul className='flex-column'>
-          {this.props.todos.map((todo) => (
-            <TodoItem
-              key={todo.id}
-              todo={todo}
-              toggleTodo={this.props.toggleTodo}
-              handleDeleteTodo={this.props.handleDeleteTodo}
-            />
-          ))}
+          { this.props.isCompleted === true ?
```
Comment (3ba2ii) [inline week_2/practice/src/components/TodoList.js:10]: Overall Perfect 👌🏻

**Review by 3ba2ii — COMMENTED** [review]
I've added a review for each section of your PR, Great job Mina 👏🏻

File: week_2/practice/src/components/TodoList.js (hunk 3 above)
Comment (MinaKamil) [reply week_2/practice/src/components/TodoList.js:10]: Thanks for your feedback ☺️

File: week_2/practice/src/components/TodoList.js (hunk 2 above)
Comment (MinaKamil) [reply week_2/practice/src/components/TodoList.js:8]: Yes , this is better for ux
Thank you very much

**Pull request closed without merging**